packages/db/src/types/execution.ts

```diff
@@ -1,10 +1,20 @@
+import { ExecutionRecordStatus } from '@cumulus/types/api/executions';
+
 export interface PostgresExecution {
   arn: string,
-  status: string,
   async_operation_cumulus_id?: number,
   collection_cumulus_id?: number,
   parent_cumulus_id?: number,
   cumulus_version?: string,
+  url?: string,
+  status?: ExecutionRecordStatus,
+  tasks?: Object, // TODO need specific type?
+  error?: Object, // TODO need specific type?
```

Comment:
Do/should we have specific types for these? Are there keys we know will always be there?

Reply:
I could be wrong, but taking a look at the [model](https://github.com/nasa/cumulus/blob/master/packages/api/models/executions.js#L52-L53), there doesn't seem to be specific types for these.

Reply:
Ok, yeah that makes sense. It didn't look like we had types for these. Just wasn't sure if I missed 'em. Thanks for the sanity check 😄